### ISSUE TITLE
The loss cannot converge

### ISSUE THREAD

**Issue by vuonglq-vn** [Guanghan/GNet-pose#6]
Hi @Guanghan ,
Thanks for your great project.
Can you show to me your loss curve of your final model?
Below is my result when training GNet-pose with base_lr: 0.0001, batch_size: 4, after 3 days, trained 76 epochs.
![313](https://user-images.githubusercontent.com/11450887/40895781-50ab5818-67ec-11e8-9774-c902aff285d2.png)
It still cannot converge.
If you trained with other parameter in prototxt file, please tell me.
I'm waiting for your feedback.

**Comment by strawberryfg** [comment]
Hi @lelordoftech  I was wondering what's your PCKh on train & val of MPII? I have tried 2-stage Hourglass (original Hourglass) with pretrained caffemodel weights provided by the author, and achieved only around 80%PCKh on val. I started with learning rate 2.5e-5 w/ RMSProp. The loss cannot converge when I started from 2.5e-4 w/ or w/o pretrained weights, which I found sort of weird. My batch size is 3 on a single GPU. Thanks!
